aws/table_aws_batch_job.go

```diff
@@ -0,0 +1,312 @@
+package aws
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/batch"
+	// "github.com/aws/aws-sdk-go-v2/service/batch/types"
+
+	batchv1 "github.com/aws/aws-sdk-go/service/batch"
+
+	"github.com/turbot/steampipe-plugin-sdk/v5/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v5/plugin/transform"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsBatchJob(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_batch_job",
+		Description: "AWS Batch Job",
+		Get: &plugin.GetConfig{
+			KeyColumns: plugin.SingleColumn("name"),
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: shouldIgnoreErrors([]string{"InvalidParameterValueException"}),
+			},
+			Hydrate: getBatchJob,
+		},
+		List: &plugin.ListConfig{
+			Hydrate: listAwsBatchJobs,
+		},
+		GetMatrixItemFunc: SupportedRegionMatrix(batchv1.EndpointsID),
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "arn",
+				Description: "The Amazon Resource Name (ARN) of the job.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("JobArn"),
+			},
+			{
+				Name:        "job_id",
+				Description: "The job ID.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "name",
```

Comment:
Can you please rename the column name to `job_name` instead? We should keep the column names as per the API response except the ARN column.